packages/datadog-plugin-protobufjs/test/schemas/other_message.proto

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I think the newline should be removed:
https://protobuf.dev/programming-guides/proto3/#simple

Reply:
Fixed this